handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslServerContext.java

```diff
@@ -206,4 +220,22 @@ void verify(ReferenceCountedOpenSslEngine engine, X509Certificate[] peerCerts, S
             manager.checkClientTrusted(peerCerts, auth, engine);
         }
     }
+
+    private static final class OpenSslSniHostnameMatcher implements SniHostNameMatcher {
+        private final OpenSslEngineMap engineMap;
+
+        OpenSslSniHostnameMatcher(OpenSslEngineMap engineMap) {
+            this.engineMap = engineMap;
+        }
+
+        @Override
+        public boolean match(long ssl, String hostname) {
```

Comment:
Not related with this PR but shouldn't this be `matches` for consistency's sake?